syntax = "proto2";

package example_msgs;

message TestRequest {
  required uint32 a = 1;
  required uint32 b = 2;
}

message TestResponse {
  required uint32 sum = 1;
}

